hooks/jenkins/jira-workflow/Jenkinsfile

```diff
@@ -50,23 +43,22 @@ pipeline {
         script {
           try {
             settings = readYaml(file: '.github/jira-workflow.yml')
-            //sh("echo ${settings.project}")
           } catch(err) {
             echo "Please create .github/jira-workflow.yml"
             throw err
-            //currentBuild.result = 'ABORTED'
-            //return
-            //currentBuild.rawBuild.result = Result.ABORTED //This method requires in-process script approval, but is nicer than what's running currently
           }
         }
       }
     }
     stage('Get project info') {
       steps {
         script {
-          //  echo projectId
           projectInfo = jiraGetProject(idOrKey: projectId, site: 'Jira')
-          //  echo projectInfo.data.name.toString()
+          jql = jiraJqlSearch(jql: "fixVersion=${name}", site: 'Jira') // Query Jira for issues related to the release
+          releaseDescription = "### Release Notes\\n${description}\\n### Associated Jira Issues\\n"
```

Comment:
This needs to be on one line, since we're using JSON to post... we'll create newlines based on `\\n` in the string